artifactory/commands/transferfiles/transferfileprogress.go

```diff
@@ -222,6 +241,17 @@ func (t *TransferProgressMng) RemoveRepository() {
 	time.Sleep(progressbar.ProgressRefreshRate)
 }
 
+func (t *TransferProgressMng) RemoveFile(chunkSize int) {
```

Comment:
The name of this function seem wrong and unclear. By reading the name of this function, the reader can this that it actually removes a file. 
Actually, this function seem to be unused. Should it be removed?